packages/webapp/pages/popup/notifications/enable.tsx

```diff
@@ -0,0 +1,94 @@
+import classNames from 'classnames';
+import BrowserPermissionIcon from '@dailydotdev/shared/src/components/icons/BrowserPermission/primary.svg';
+import NotificationToggleIcon from '@dailydotdev/shared/src/components/icons/NotificationToggle/primary.svg';
+import { BootDataProvider } from '@dailydotdev/shared/src/contexts/BootProvider';
+import { useNotificationContext } from '@dailydotdev/shared/src/contexts/NotificationsContext';
+import classed from '@dailydotdev/shared/src/lib/classed';
+import { postWindowMessage } from '@dailydotdev/shared/src/lib/func';
+import React, { useEffect } from 'react';
+import { ENABLE_NOTIFICATION_WINDOW_KEY } from '@dailydotdev/shared/src/hooks/useNotificationPermissionPopup';
+import { BootApp } from '@dailydotdev/shared/src/lib/boot';
+
+const getRedirectUri = () =>
+  `${window.location.origin}${window.location.pathname}`;
+
+const InstructionContainer = classed(
+  'div',
+  'flex flex-col items-start p-4 mt-5 rounded-8 border border-theme-divider-tertiary',
+);
+
+const iconClasses = 'flex flex-grow mt-2';
+
+const Description = classed('p', 'typo-callout text-theme-label-tertiary');
```

Comment:
We should probably generalize this app wide, but no need to fix it for now.
Perhaps you can log it as a ticket in the backlog.

Reply:
I agree. We will need to come up with a very intuitive naming based on our guidelines. Let me log it quickly.